rdkit/Chem/PandasTools.py

```diff
@@ -332,7 +338,8 @@ def LoadSDF(filename, idName='ID', molColName='ROMol', includeFingerprints=False
         close = None  # don't close an open file that was passed in
     records = []
     indices = []
-    for i, mol in enumerate(Chem.ForwardSDMolSupplier(f, sanitize=(molColName is not None))):
+    for i, mol in enumerate(Chem.ForwardSDMolSupplier(f, sanitize=(molColName is not None),
```

Comment:
Seems like it would be nicer to explicitly include a sanitize option?

Reply:
Agreed, although I’m not entirely sure what the current logic is.

Reply:
yeah, I had to look that up. That's there so that you can read the properties out of the file efficiently. you don't end up with a molecule column at all.